src/main/java/org/terasology/blockNetwork/TwoNetworkNodes.java

```diff
@@ -18,11 +18,27 @@
 public class TwoNetworkNodes {
     public final NetworkNode node1;
     public final NetworkNode node2;
+    
+    /**
+    * TwoNetWorkNodes constructor.
+    *
+    * @param node1 
+    * @param node2
+    */
 
     public TwoNetworkNodes(NetworkNode node1, NetworkNode node2) {
         this.node1 = node1;
         this.node2 = node2;
     }
+    
+    /**
+    * Tells if two nodes are equal to each other and indicates if object o is null.
```

Comment:
"Checks of two nodes are equal" should be enough. No need to mention the null thing, since the function doesn't specifically indicate that an object is null. It just checks if two objects are the same and returns true or false accordingly